events/triggers/debug

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Function to upload the output to hastebin
+# Source: https://github.com/seejohnrun/haste-client#lightweight-alternative
+haste() { a=$(cat); curl -X POST -s -d "$a" https://hastebin.com/documents | awk -F '"' '{print "https://hastebin.com/"$4}'; }
+
+UMBREL_ROOT="$(readlink -f $(dirname "${BASH_SOURCE[0]}")/../..)"
+DEBUG_FILE="${UMBREL_ROOT}/scripts/debug"
+
+if [ ! -f ${DEBUG_FILE} ]; then
+  wget https://raw.githubusercontent.com/getumbrel/umbrel/master/scripts/debug -O ${DEBUG_FILE} && chmod +x ${DEBUG_FILE}
+fi
```

Comment:
When can this condition be true? If the user deletes files, we shouldn't have to have code for that specific case.

Reply:
For example if the user doesn't run the latest Umbrel version and thus the script is not included.

Reply:
But then this trigger wouldn't exist either.

Reply:
Yeah, seen like this 😂

Reply:
Resolved in 589207a.